Exiled.API/Features/Room.cs

```diff
@@ -276,10 +276,10 @@ private static RoomType FindType(string rawName)
 
         private static ZoneType FindZone(GameObject gameObject)
         {
-            var transform = gameObject.transform;
+            Transform transform = gameObject.transform;
 
             if (transform.parent == null)
-                return ZoneType.Unspecified;
```

Comment:
Does the surface room object not have a transform.parent anymore? 
Because this could mess with rooms that are generated off the map, like NW's debugging rooms that sometimes make it into builds, if they decide to keep them around. 
Or if someone somehow makes a new room object for a room that doesn't exist.

Reply:
I'm gonna send you the list of rooms, transforms and parents on Discord.